kedro-datasets/pyproject.toml

```diff
@@ -232,7 +232,7 @@ test = [
     # huggingface
     "datasets",
     "huggingface_hub",
-    "transformers",
+    "transformers[torch]",
```

Comment:
Previously `transformers` (Hugging Face) was using TensorFlow (since it's installed as part of the tests), but PyTorch support is more robust